src/messenger-shortcuts.js

```diff
@@ -129,7 +129,8 @@ function focusMessageInput() {
 }
 
 function selectFirstSearchResult() {
-  var first = document.querySelector('ul[role="listbox"] li div');
+  var contacts = document.querySelectorAll('ul[role="listbox"]')[1];
```

Comment:
When you only type a single letter into the search box, the "Search Messages" section doesn't show up, so the contacts `listbox` is the first section.